logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -80,6 +84,11 @@ type ConfLogger struct {
 
 // Init does nothing
 func (lm *Plugin) Init() error {
+	// Environmental variable is preferred before config file. If set, all logger will be created with its log level
+	// and config file processing can be skipped
+	if os.Getenv("INITIAL_LOGLVL") != "" {
+		return nil
```

Comment:
This is not the correct solution to the previously discussed issue with precedence.

The environmental variable `INITIAL_LOGLVL` is only used to change global variable [initialLogLvl](https://github.com/ligato/cn-infra/blob/master/logging/logrus/registry.go#L31) at the earliest time possible. The `initialLogLvl` variable is used for in registries as **initial level** used when some logger doesn't have level set explicitly. However we still want to be able to set level for specific loggers.

The correct way would be to put this condition only for using the default level from configuration, so it doesn't override the environmental variable. The other part of configuration file is defining level for specific loggers and that takes precedence over the initial level of loggers. If we had environmental variable for specific logger levels, we would use it there.